src/dispatch/plugins/dispatch_slack/views.py

```diff
@@ -221,7 +221,7 @@ async def handle_command(
         )
 
         # allow everyone other than participants engage in sensitive commands
-        if participant.role == ParticipantRoleType.participant:
+        if participant.current_role.role == ParticipantRoleType.participant:
```

Comment:
Heads up that participants can have more than one current role and you could be comparing a list or roles with a string here.

Reply:
Good call, I've updated my logic.